### PR TITLE
fix(responsive): docs site

### LINKED ISSUE #267: Responsive Styles/Components Demo Site
Our responsive behavior on the demo site seems to have broken due to updates to the tabs and select components.  I'll assign this one to myself to get this fixed.

screen shot 2018-05-23 at 2 41 31 pm

### PR DESCRIPTION
**Pull request by andrew-frueh**
fixes responsive behavior on the docs site that had broken over time previous updates

@benjanderson and I noticed that the app switcher functionality is broken on the navbar mobile menu.  So that has been commented out until Ben has a chance to take a look at it.  I'll reopen the issue where we thought we fixed. 

closes #267

### CODE REVIEW

File: projects/cashmere/src/lib/navbar/navbar-mobile-menu/navbar-mobile-menu.component.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-    <div class="bottomSpacer"></div>
-    <a *ngIf="appSwitcher=='true' && appSwitcherService?.allApplicationsUri" class="app-switcher" href={{appSwitcherService?.allApplicationsUri}}>
-        <hc-icon fontSet="fa" fontIcon="fa-th"></hc-icon>
-        <span>View all my applications</span>
-    </a>
-</div>
+<div class="menu-dropdown" [style.top.%]="_yPos">
+    <ng-content></ng-content>
+    <div class="bottomSpacer"></div>
+    <!--<a *ngIf="appSwitcher=='true' && _appSwitcherService?.allApplicationsUri" class="app-switcher" href={{_appSwitcherService?.allApplicationsUri}}>
```
Comment (jtrujill) [inline projects/cashmere/src/lib/navbar/navbar-mobile-menu/navbar-mobile-menu.component.html]: I don't know what the plan is with this and I wont say it's a deal breaker but I hate commented out code. It's best to delete and refer to the git history if needed. If this is temp than I'd say it's okay

Comment (andrew-frueh) [reply projects/cashmere/src/lib/navbar/navbar-mobile-menu/navbar-mobile-menu.component.html]: Yeah - Ben and I just weren't able to fix it on the phone this morning.  But you reminded me that I need to reopen that issue.  I'll do that now

File: projects/cashmere/src/lib/navbar/navbar-mobile-menu/navbar-mobile-menu.component.ts [hunk 2]
```diff
@@ -29,6 +27,6 @@ export class NavbarMobileMenuComponent {
 
     /** Hide the component from view */
     hide(): void {
-        this._yPos = '-100';
+        this._yPos = '-50';
```
Comment (jtrujill) [inline projects/cashmere/src/lib/navbar/navbar-mobile-menu/navbar-mobile-menu.component.ts]: any reason this isn't being removed or display:none?

Comment (andrew-frueh) [reply projects/cashmere/src/lib/navbar/navbar-mobile-menu/navbar-mobile-menu.component.ts]: It's for the animation to slide up/down

File: src/app/components/components.component.ts [hunk 3]
```diff
@@ -1,14 +1,31 @@
...
     docItems: DocItem[];
+    thisPage = '';
+    selectOptions: Array<string> = [];
 
-    constructor(docItemService: DocumentItemsService) {
+    constructor(docItemService: DocumentItemsService, private activatedRoute: ActivatedRoute, private router: Router) {
         this.docItems = docItemService.getDocItems();
+
+        // Listen for vertical tab bar navigation and update the select component
+        router.events.subscribe(event => {
```
Comment (jtrujill) [inline src/app/components/components.component.ts]: You'll need to unsubscribe from the observable

Comment (andrew-frueh) [reply src/app/components/components.component.ts]: Thanks for walking me through this fix @jtrujill - let me know if the updates were implemented correctly

**Pull request merged**